### PR TITLE
Add tests for PrimitiveFloatArraySubject empty/null and out-of-range exact-equality cases

### PR DESCRIPTION
**Pull request by vasiliy-mikhailov**
Adds 8 tests for `PrimitiveFloatArraySubject` covering isEmpty/isNotEmpty success and failure paths, null-array failures for `usingExactEquality` and `usingTolerance`, and the IllegalArgumentException messages produced when an out-of-range negative int or long is passed to `usingExactEquality().contains()`. Tests use Truth's own assertThat/expectFailure idiom and assert on concrete failure facts.

| metric | before | after |
|---|---|---|
| mutation score | 79% | 97% |
| test methods added | n/a | 8 |

The additions are append-only (no existing test is modified) and pass against the current code.

---

**How this was produced**

This PR was generated with an AI-assisted pipeline built around mutation testing (PIT). The pipeline mutates the target class (flipping conditions and changing boundary/edge cases) and runs the existing tests against each mutant. Where a mutant survives (the existing tests do not catch that edge case), it writes a focused test for that case and reruns PIT to confirm the new test actually kills that specific mutant. So every added test is verified to catch a concrete edge case the suite missed before, rather than being speculative or redundant. The change is additive only (no production code modified), and the module builds green under its CI JDK.

### CODE REVIEW

File: core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java [hunk 1]
```diff
@@ -833,4 +833,86 @@ public void hasLengthNullArray() {
...
+
+  @Test
+  public void usingTolerance_nullArray_failure() {
+    expectFailure(
+        whenTesting ->
+            whenTesting.that((float[]) null).usingTolerance(0.001).contains(1.0f));
+  }
+
+  @Test
+  public void usingExactEquality_contains_otherTypes_negativeIntOutOfRange() {
```
Comment (chaoren) [inline core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java:872]: Let's move these next to the other `usingExactEquality_` tests.

File: core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java [hunk 2]
```diff
@@ -833,4 +833,86 @@ public void hasLengthNullArray() {
...
+
+  @Test
+  public void usingExactEquality_nullArray_failure() {
+    expectFailure(
+        whenTesting ->
+            whenTesting.that((float[]) null).usingExactEquality().contains(1.0f));
+  }
+
+  @Test
+  public void usingTolerance_nullArray_failure() {
```
Comment (chaoren) [inline core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java:865]: Let's move this next to the other `usingTolerance_` tests.

**Review by chaoren — APPROVED** [review]

File: core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java (hunk 1 above)
Comment (chaoren) [reply core/src/test/java/com/google/common/truth/PrimitiveFloatArraySubjectTest.java:872]: We can move these ourselves after the tests are imported.

**Comment by chaoren** [comment]
Thank you for the tests!
